pkg/cloudprovider/createfleetbatcher.go

```diff
@@ -107,15 +107,15 @@ func (b *CreateFleetBatcher) run() {
 }
 
 func (b *CreateFleetBatcher) waitForIdle() {
```

Comment:
Possible to reuse our batcher code?

Reply:
I'm not sure what you mean

Reply:
https://github.com/aws/karpenter-core/blob/main/pkg/controllers/provisioning/batcher.go#L27

Reply:
ah yeah, I have some work to abstract out the batcher and use for TerminateInstances as well, so will post in a follow-up.